metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/force_delete_test.clj

```diff
@@ -66,3 +75,82 @@
        ;; tag association associated with other collection revision or whole collection is not deleted
        (util/is-tag-association-deleted? ta1 false)
        (util/is-tag-association-deleted? ta3 false)))))
+
+(deftest force-delete-service
+  (let [coll (util/create-and-save-collection "REG_PROV" 1)
+        coll-concept-id (:concept-id coll)
+        misc-svc (util/create-and-save-service "REG_PROV" 42)
+        misc-svc-assn (util/create-and-save-service-association
+                       coll misc-svc 1)
+        expected-svc-concept-id "S1200000003-REG_PROV"
```

Comment:
These concept-ids can change in the future for changes unrelated to services. Anytime you want to specifically verify a concept-id you should specify the concept-id in the creation call.

Reply:
Done.